ingestion_tools/scripts/data_validation/conftest.py

```diff
@@ -136,58 +127,12 @@ def pytest_configure(config: pytest.Config) -> None:
     config.addinivalue_line("markers", "voxel_spacing: Tests concerning voxel spacings.")
 
 
-# @pytest.hookimpl(hookwrapper=True)
-# def pytest_runtest_makereport(item, call):
-#     # pytest.set_trace()
-#     # yield
-#     has_epic = False
-#     has_feature = False
-#     has_story = False
-
-#     paramnames = []
-#     params = {}
-
-#     epic = None
-#     feature = None
-#     story = None
-
-#     for m in item.iter_markers():
-#         if m.name == ALLURE_LABEL_MARK:
-#             if m.kwargs.get("label_type") == LabelType.EPIC:
-#                 has_epic = True
-#                 epic = m
-#             if m.kwargs.get("label_type") == LabelType.FEATURE:
-#                 has_feature = True
-#                 feature = m
-#             if m.kwargs.get("label_type") == LabelType.STORY:
-#                 has_story = True
-#                 story = m
-
-#         # if m.name == "parametrize":
-#         #     paramnames = m.args[0].strip().split(",")
-#         #     ptup = m.args[1][0]
-#         #
-#         #     if len(paramnames) == 1:
-#         #         params[paramnames[0].strip()] = ptup
-#         #     else:
-#         #         params = {pn.strip(): p for pn, p in zip(paramnames, ptup)}
-
-#     if "dataset" in item.fixturenames:
-#         dataset = pytest.dataset
-#         epic_name = f"Dataset {dataset}"
-#         mark = getattr(pytest.mark, ALLURE_LABEL_MARK)(epic_name, label_type=LabelType.EPIC)
-#         item.add_marker(mark)
-
-#     if "run_name" in item.fixturenames:
-#         run_name = item.callspec.params["run_name"]
-#         feature_name = f"Run {run_name}"
-#         mark = getattr(pytest.mark, ALLURE_LABEL_MARK)(feature_name, label_type=LabelType.FEATURE)
-#         item.add_marker(mark)
-
-#     if "voxel_spacing" in item.fixturenames:
-#         voxel_spacing = item.callspec.params["voxel_spacing"]
-#         story_name = f"Voxel spacing {voxel_spacing}"
-#         mark = getattr(pytest.mark, ALLURE_LABEL_MARK)(story_name, label_type=LabelType.STORY)
-#         item.add_marker(mark)
-
-#     yield
+@pytest.hookimpl(hookwrapper=True)
+def pytest_runtest_makereport(item: pytest.Item):
+    if "voxel_spacing" in item.fixturenames:
+        allure.dynamic.story(f"VoxelSpacing {item.callspec.params['voxel_spacing']}")
+    if "run_name" in item.fixturenames:
+        allure.dynamic.feature(f"Run {item.callspec.params['run_name']}")
+    if "dataset" in item.fixturenames:
+        allure.dynamic.epic(f"Dataset {pytest.dataset}")
+    yield
```

Comment:
yes, all of the past code be just simplified to this, which is doing the exact same thing